### ISSUE TITLE
No code sample for 11.3.4

### ISSUE THREAD

**Issue by phderome** [ReactiveDesignPatterns/CodeSamples#5]
Section 11.3.4 of pBook advertises source code archives for details on usage of CallingThreadDispatcher, which does not look to be present here.

**Comment by rkuhn** [comment]
Yes, the bad conscience finally came back to haunt me: this slipped through the while writing. I’ll think of something to show, there are a few quite days ahead.

**Comment by phderome** [comment]
Btw, you may have duplicate code to consider: the available zip on Manning's site and the github repo; personally I am more interested in the github version.

**Comment by rkuhn** [comment]
The ZIP file served by Manning can be updated as improvements are made here, it is just an archive of a tagged version.
